apps/src/craft/simple/craft.js

```diff
@@ -610,6 +603,12 @@ Craft.executeUserCode = function () {
 
   var appCodeOrgAPI = Craft.gameController.codeOrgAPI;
   appCodeOrgAPI.startCommandCollection();
+  appCodeOrgAPI.registerEventCallback(null, event => {
+    if (event.eventType === EventType.WhenUsed && event.targetType === 'sheep') {
+      appCodeOrgAPI.drop(null, 'wool', event.targetIdentifier);
+    }
+  });
```

Comment:
Sorry for not fixing this before merging — opened https://github.com/code-dot-org/code-dot-org/pull/17260 to address!